### PR TITLE
Wu7e9auF: Update Billing Recorder to populate event_id column on billing_events

### PR DESCRIPTION
**Pull request by chrisclayson**
## What

The `billing_events` table current has no primary key defined. It has been determined that we should ideally use the event_id for this. Unfortunately, event_id is not currently stored in this table.

This card is to update the billing recorder service to populate the `event_id` column when creating new `billing_events`.

## Why

Storing the `event_id` will enable efficient joins to both the `audit_events` table and the new `billing_status` table.

## How

Update tests to check for `event_id` value
Add `event_id` column to `INSERT` statement in `database.write_billing_event_to_database`
Also updated `run-test.sh` to tell Docker not use layer cache as it was using cached version of Postgres image rather than pulling latest migration scripts.

### CODE REVIEW

File: build/run-tests.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-docker-compose build
+docker-compose build --no-cache
```
Comment (alan-gds) [inline build/run-tests.sh]: This looks like it ideally belongs in a separate commit?

**Comment by chrisclayson** [comment]
[Update Billing Recorder to populate event_id column on billing_events.](https://trello.com/c/Wu7e9auF/164-update-billing-recorder-to-populate-eventid-column-on-billingevents)

File: build/run-tests.sh (hunk 1 above)
Comment (alexandergeegds) [reply build/run-tests.sh]: Or removed? Why do we want to prevent use of the cache?

Comment (chrisclayson) [reply build/run-tests.sh]: When I was building, it was using a cached version of the layer that does the `git clone`, so I was never pulling the latest version of the database migrations.

Probably need to investigate why, but in the short term, this was the fix.

Comment (chrisclayson) [reply build/run-tests.sh]: A good explanation of how layer caching works and why I was seeing the problem here: https://thenewstack.io/understanding-the-docker-cache-for-faster-builds/

Comment (alexandergeegds) [reply build/run-tests.sh]: 😞 I thought Docker was more intelligent than that... 😢
